### DISCUSSION TITLE
Add Daycore mod from McOsu

### DISCUSSION THREAD

**Discussion by Leclowndu93150** [ppy/osu#20669] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/73310949/194776599-a26422e1-b77a-42c6-984e-269e2ffdb468.png)

it changes the voice of the song to make a deep voice

**Comment by 12beesinatrenchcoat** [comment, marked as answer]
The already existing Daycore mod seems to be functionally the same.
![osu!_VhTkNLWxeN](https://user-images.githubusercontent.com/25379179/194777493-a68d2b4c-532d-4b69-a5cb-f16b8924c8f3.png)
